app/jobs/sync/bitflyer/update_missing_deposits_job.rb

```diff
@@ -0,0 +1,9 @@
+class Sync::Bitflyer::UpdateMissingDepositsJob < ApplicationJob
+  queue_as :low
+
+  def perform
+    Publisher.joins(:bitflyer_connection).joins(:channels).where(selected_wallet_provider: BitflyerConnection.class.name.to_s).where( channels: { deposit_id: nil}).select("channels.id").each do |channel_id|
```

Comment:
Can we add this as a `self` method on the `Publisher` model? That way we can say something like `Publisher.blank_biflyer_deposit_ids`and keep DB access details in that file.

Reply:
I don't expect the scope to be re-used, but I think the lack of indentation makes it hard to read, which I will fix.